tools/Utils/c_rehash.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+
+import os
+import argparse
+import subprocess
+
+def process_file (dirname, file) :
+    # Old full file ocation
+    full_path = dirname + "\\" + file
```

Comment:
@Kozoriz 
Looks like python has its own separator function for path (instead of \\)